SYCL/GroupAlgorithm/SYCL2020/sort.cpp

```diff
@@ -0,0 +1,361 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -I . -o %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+//
+// Missinsg __spirv_GroupIAdd, __spirv_GroupSMin and __spirv_GroupSMax on AMD:
+// XFAIL: rocm_amd
+
+// TODO: enable compile+runtime checks for operations defined in SPIR-V 1.3.
```

Comment:
I believe spirv-max-version was already set to 1.3 long ago: https://github.com/intel/llvm/commit/0a67b10f2ca0d32be5d08938160422527c42e5f9

Reply:
ok. Remove it. Just "copy-paste" issue